presto-main/src/main/java/com/facebook/presto/operator/WorkProcessor.java

```diff
@@ -77,14 +77,18 @@
         return WorkProcessorUtils.flatTransform(this, transformation);
     }
 
+    /**
+     * Transforms {@link WorkProcessor} using {@link Transformation}. {@link Transformation} instance will be disposed immediately after
```

Comment:
see https://github.com/prestodb/presto/pull/10767#discussion_r194169560

Reply:
Since this commit is already being reviewed there & this is not a preparatory commit in this PR, let's remove it from here.

(thus, i am not reviewing it here)